.storybook/util/story-helpers.js

```diff
@@ -1,4 +1,16 @@
-import { get } from 'lodash/fp';
+function get(obj, paths, defaultValue) {
```

Comment:
Can't we replace `_.get()` in this case with native property access like `obj.path`? This looks overly complicated, but I might be missing something

Reply:
Not quite. We have an object and an array of path segments to a nested property. You need to iterate over the path segments somehow to get to the property. Granted, this function does a bit more with error handling in case the path doesn't exist and a default value.